examples/RM3NewPTOSim/RM3Files/README.md

```diff
@@ -0,0 +1,11 @@
+# PTO-Sim Compressible Hydraulic PTO
+
+**Author:**  	Ratanak So
```

Comment:
@jleonqu Minor comment, but I would expect you should be listed as an additional Author of this example or repository.